spec.bs

```diff
@@ -667,7 +674,33 @@ picture-in-picture experience. Use the
 button.
 
 <pre class="lang-javascript">
-documentPictureInPicture.requestWindow({ disallowReturnToOpener: true });
+documentPictureInPicture.requestWindow({
+  disallowReturnToOpener: true
+}).then(...);
```

Comment:
Maybe use `await` as `.then(...);` is not valid JavaScript.

Reply:
because of the ellipsis?  the intent of "do whatever you want" seemed much clearer to me with `...` than the more concrete but arbitrary `()=>{}`.

i'll switch to `await`.